test/infra/control/test-ci-mysqlx-dbdeployer-install.bash

```
@@ -0,0 +1,24 @@
+  capture { print }
+  capture && /^      - name: Download and unpack MySQL 8\.4$/ { exit }
+' "${workflow}")"
+
+guard_body="$(printf '%s\n' "${step}" | awk '
+  /if ! command -v dbdeployer >\/dev\/null 2>&1; then/ { capture = 1 }
+  capture { print }
+  capture && /^          fi$/ { exit }
+')"
+
+printf '%s\n' "${guard_body}" | grep -Fqx '          if ! command -v dbdeployer >/dev/null 2>&1; then'
+printf '%s\n' "${guard_body}" | grep -Fqx '            curl --fail --location --proto '\''=https'\'' --tlsv1.2 \\'
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)